### ISSUE TITLE
[FEAT] Mailbox folder split geographicaly

### ISSUE THREAD

**Issue by stdevPavelmc** [stdevPavelmc/mailad#183]
Some [big & important] users are requesting an optional mailbox isolation on folders. I explain...

They want [for a particular reason] that the user's mailboxes of the domain (~10k users) is split on geographical bases, like

```
/home/vmail/
                   /habana
                   /pinar
                   /isla
                   /matanzas
                   /noexiste ;-)
                   /villaclara
...
```

That's doable and not hard, please comment if you need a feature like this.

**Comment by stdevPavelmc** [comment]
Giving this a quick round...

0. It makes sense to use the property ` office` aka "Office" see the picture it's the "Oficina" one: ![User details](https://github.com/stdevPavelmc/mailad/blob/master/imgs/admin_user_details.png?raw=true)
1. It's on the same properties page, so no extra clicking to find "an obscure property down there..."
2. It must be an opt-in feature, off by default to maintain compatibility an respect sysadmins using the Office property now.

**Comment by stdevPavelmc** [comment]
This will be a opt-in feature, aka: it will be disabled by default.

So you have to enable it by hand, the sub-folder will be in the Office property of the Active Directory

Working on it.

**Comment by stdevPavelmc** [comment]
Closed by PR #187
